packages/shared/src/lib/core/network/utils/isSupportedNetworkId.ts

```diff
@@ -0,0 +1,6 @@
+import { SupportedNetworkId } from '../enums'
+import { NetworkId } from '../types'
+
+export function isSupportedNetworkId(networkId: NetworkId): boolean {
```

Comment:
dup0licate logic on isOfficialNe
```suggestion
export function isSupportedNetworkId(networkId: NetworkId): boolean {
```twork?